### PR TITLE
updated chart size initialization that was taken default values

### PR DESCRIPTION
**Pull request by pnibaldi**
Before chart was taking default w=600px and height=400px which was truncating the svg generated to export.
Need to have back end changes deploy fo the export to work properly.

### CODE REVIEW

File: rd_ui/app/scripts/controllers/dashboard.js [hunk 1]
```diff
@@ -78,6 +78,8 @@
         if (data.data.length) {
           $http({
             url: '/services/pdf/create',
+            //local dev url
+            //url: '172.16.15.120:12345/create',
```
Comment (mmelograno) [inline rd_ui/app/scripts/controllers/dashboard.js]: I think this should be removed

**Comment by mmelograno** [comment]
Reviewed and able to merge.

File: rd_ui/app/scripts/controllers/dashboard.js (hunk 1 above)
Comment (pnibaldi) [inline rd_ui/app/scripts/controllers/dashboard.js]: I had left it there for local use, but I can remove it :)

Comment (mmelograno) [inline rd_ui/app/scripts/controllers/dashboard.js]: np

**Pull request merged**